packages/web/hooks/use-feature-flags.ts

```diff
@@ -26,10 +27,33 @@ export type AvailableFlags =
   | "topAnnouncementBanner"
   | "tfmProTradingNavbarButton";
 
-type ModifiedFlags =
-  | Exclude<AvailableFlags, "mobileNotifications">
-  | "_isInitialized"
-  | "_isClientIDPresent";
+type OptionalFlags = "_isInitialized" | "_isClientIDPresent";
+type RequiredFlags = Exclude<AvailableFlags, "mobileNotifications">;
+
+type ModifiedFlags = RequiredFlags & Partial<Record<OptionalFlags, boolean>>;
+
+const defaultFlags: Record<ModifiedFlags, boolean> = {
+  concentratedLiquidity: true,
+  staking: true,
+  swapsAdBanner: true,
+  notifications: true,
+  convertToStake: true,
+  upgrades: true,
+  tokenInfo: true,
+  newAssetsTable: false,
+  sidebarOsmoChangeAndChart: true,
+  multiBridgeProviders: true,
+  unlistedAssets: false,
+  earnPage: false,
+  sidecarRouter: true,
+  legacyRouter: true,
+  tfmRouter: true,
+  osmosisUpdatesPopUp: false,
+  aprBreakdown: true,
+  newPoolsTable: true,
+  topAnnouncementBanner: true,
+  tfmProTradingNavbarButton: true,
```

Comment:
How should we decide what is default true, should we not just have all be true?

Reply:
🤷. I'm ok with them all being true.